package.json

```diff
@@ -23,36 +13,35 @@
     "encoding",
     "litecoin"
   ],
+  "homepage": "https://github.com/bitcoinjs/bs58check",
+  "bugs": {
+    "url": "https://github.com/bitcoinjs/bs58check/issues"
+  },
+  "license": "MIT",
+  "author": "Daniel Cousens",
+  "files": [
+    "index.js"
+  ],
   "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bitcoinjs/bs58check.git"
+  },
   "scripts": {
     "standard": "standard",
-    "coverage": "mocha --require blanket -R travis-cov",
-    "coverage-local": "mocha --require blanket -R html-cov",
+    "coverage": "npm run coverage-local && nyc report --reporter=text-lcov |coveralls",
+    "coverage-local": "nyc npm run unit",
     "test": "npm run standard && npm run unit",
-    "unit": "mocha"
-  },
-  "config": {
-    "blanket": {
-      "pattern": [
-        ""
-      ],
-      "data-cover-never": [
-        "node_modules",
-        "test"
-      ]
-    },
-    "travis-cov": {
-      "threshold": 100
-    }
+    "unit": "tape test/*.js"
   },
   "dependencies": {
     "bs58": "^2.0.1",
     "create-hash": "^1.1.0"
   },
   "devDependencies": {
-    "blanket": "*",
-    "mocha": "*",
-    "standard": "*",
-    "travis-cov": "*"
+    "coveralls": "^2.11.9",
+    "nyc": "^6.1.1",
```

Comment:
``` bash
time npm i blanket
...
real    0m3.810s
user    0m1.123s
sys     0m0.113s
```

``` bash
time npm i standard
...
real    0m35.819s
user    0m6.267s
sys     0m0.723s
```

``` bash
time npm i nyc
real    0m49.740s
user    0m6.183s
sys     0m0.500s
```

I agree, `standard` takes too long (halp @feross).
This doesn't mean I want to *double* that time with `nyc`.